lib/screens/components/password_status_banner.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
```

Comment:
Add copyright comment on top of new files 
// Copyright (c) Silence Laboratories Pte. Ltd.
// This software is licensed under the Silence Laboratories License Agreement.